source/documentation/administration_guide/topics/proc_migrating_she_host_to_different_cluster.adoc

```diff
@@ -0,0 +1,30 @@
+// module included:
+// doc-Administration_Guide/chap-Hosts.adoc
+
+[id='migrating_she_host_to_different_cluster_{context}']
+= Migrating a self-hosted engine host to a different cluster
+
+You cannot migrate a host that is configured as a self-hosted engine host to a data center or cluster other than the one in which the self-hosted engine virtual machine is running. All self-hosted engine hosts must be in the same data center and cluster.
+
+You need to disable the host from being a self-hosted engine host by undeploying the self-hosted engine.
+
+.Procedure
+
+. Click menu:Compute[Hosts] and select the host.
+. Click menu:Management[Maintenance].
+. Once the host is in maintenance mode, click *Reinstall* with the *Hosted Engine UNDEPLOY* option selected.
```

Comment:
Suggestion (if it fits the GUI): "... click *Reinstall* and select *Hosted engine UNDEPLOY*."

Reply:
@RichardHoch 

But that sounds like two separate steps:
1. Click Reinstall
2. Select Hosted engine UNDEPLOY

Maybe:
1. Under **Reinstall**, select **Hosted Engine UNDEPLOY**.
2. Click **Reinstall**.

Reply:
Your suggestion is fine -- I don't have the UI in front of me, so use whichever (original or your suggestion) is clearest for the user.

Reply:
Done.